more/release-notes.mdx

```diff
@@ -4,6 +4,44 @@ title: Release Notes
 
 All changes and improvements to Literal AI are listed here. For changes in the SDKs, go to the [Python SDK](/python-client/development/changelog) or [TypeScript SDK](/typescript-client/development/changelog).
 
+# 0.0.608-beta (June 4, 2024)
+
+This version is compatible with:
+* [Chainlit](https://docs.chainlit.io/get-started/overview) version `1.0.504` and above.
+* [Python SDK](/python-client) version `0.0.509` and above.
+* [TypeScript SDK](/typescript-client) version `0.0.503` and above.
+
+### Breaking changes
+
+None
+
+### New features
+
+* We've adapted the Online Evaluation page for our on-premise clients who may not have a Queuing system installed
+* Interactions with filters and layouts on the dashboard are now tracked on PostHog
```

Comment:
i don't know if i'd mention that to end users ? it's mostly to our advantage and doesn't really bring them any benefit

Reply:
Fair point. I thought it would be useful for on-premise users, but I'll remove it.

Reply:
I see ! This isn't a worry as Posthog tracking isn't active for our on-premise users.